prettier.d.ts

```diff
@@ -1,9 +1,9 @@
 declare module 'prettier' {
 	interface Options {
-		organizeImportsSkipDestructiveCodeActions?: boolean;
```

Comment:
Sry i didn't mean dropping the deprecated option entirely. just when calling `languageService.organizeImports`, to only pass `mode` there, like

```ts
languageService.organizeImports(
  // ...
  mode: organizeImportsMode ?? (organizeImportsSkipDestructiveCodeActions ? 'SortAndCombine' : undefined),
  // ...
)
```

Reply:
Got it, restored and updated.